src/app.js

```diff
@@ -1,21 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = 8080;
+
+const server = http.createServer((req, res) => {
+  const normalized = new URL(req.url, `http://${req.headers.host}`);
+  const path = normalized.pathname.slice(1);
+  const hasSymbolsAfterSlash = /\/[^/]+$/.test(path);
```

Comment:
The regular expression to check for symbols after the slash may fail for paths that include more than one slash.